frontend/packages/noobaa-storage-plugin/src/components/create-backingstore-page/create-bs.tsx

```diff
@@ -538,20 +554,40 @@ const CreateBackingStoreForm: React.FC<CreateBackingStoreFormProps> = withHandle
       },
       spec: {
         type: typeNoobaaMap[provider],
+        ssl: false,
+      },
+    };
+    if (provider === 'PVC') {
+      // eslint-disable-next-line
+      bsPayload.spec['pvPool'] = {
+        numVolumes: providerDataState.numVolumes,
+        storageClass: providerDataState.storageClass,
+      };
+    } else if (externalProviders.includes(provider)) {
+      bsPayload.spec = {
+        ...bsPayload.spec,
         [providerNoobaaMap[provider]]: {
           [bucketNoobaaMap[provider]]: providerDataState.target,
           secret: {
-            name: secretName,
+            name: providerDataState.secretName,
             namespace,
           },
         },
-        ssl: false,
-      },
-    };
-
+      };
+    }
+    if (provider === 'S3 Compatible') {
```

Comment:
Move all these strings to constants and the use here.  Same applies to rest of the code.

Reply:
ack